### ISSUE TITLE
Support for new languages

### ISSUE THREAD

**Issue by pyai88** [Plachtaa/seed-vc#111]
Hello, thanks for great works. I have couple questions, though: 

1. would this method work on a new language out-of-the-box? I want to confirm to see if I used it wrong since there are some properties from the trained language get in my outputs. 
2. If it cannot, would it be better to finetune from the pretrained model or train from scratch? 
3. Would you suggest how much data is needed for a new language? How long do you train your pretrained model? (I would like to estimate the cost of training.)

Thanks in advance.

**Comment by Plachtaa** [comment]
Hi there, 
If you find the output in unseen language being accented, you may try finetuning the current checkpoint with the language you desire.
I cannot give an estimation how many hours of data is required, the only thing I suggest is to use as much as you have

**Comment by EmreOzkose** [comment]
Hi, @

@pyai88 did you train model on a new language? How many hours of data did you train?
